src/KafkaFlow.Retry/Durable/Definitions/Builders/RetryDurableEmbeddedClusterDefinitionBuilder.cs

```diff
@@ -109,15 +110,15 @@ RetryDurablePollingDefinition retryDurablePollingDefinition
                     producer => producer
                         .DefaultTopic(this.retryTopicName)
                         .WithCompression(Confluent.Kafka.CompressionType.Gzip)
-                        .WithAcks(Acks.All) // TODO: this settings should be reviewed
+                        .WithAcks(Acks.Leader)
                 )
                 .AddConsumer(
                     consumer => consumer
                         .Topic(this.retryTopicName)
                         .WithGroupId(consumerGroupId)
                         .WithBufferSize(this.retryConsumerBufferSize)
                         .WithWorkersCount(this.retryConsumerWorkersCount)
-                        .WithAutoOffsetReset(AutoOffsetReset.Latest) // TODO: this settings should be reviewed
+                        .WithAutoOffsetReset(AutoOffsetReset.Earliest)
```

Comment:
Do we have idempotency protection?

Reply:
As we talked, I'll open a issue to think about it in the next moves.